src/data/published.json

```diff
@@ -0,0 +1 @@
+{"date":"2023-10-26"}
```

Comment:
```suggestion
{"last-deploy-date":"2023-10-26"}
```

Reply:
Certainly can, but this will make the usage in React slightly awkward (`published["last-deploy-date"]` instead of `published.date`)... Maybe we rename the file to `lastPublishDate.json` or `lastDeployDate.json`?